include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/EdgeFunctionComposer.h

```diff
@@ -39,8 +39,8 @@ class EdgeFunctionComposer
 
 private:
   // For debug purpose only
-  const unsigned EFComposer_Id;
-  static inline unsigned CurrEFComposer_Id = 0;
+  const unsigned EFComposerId;
+  static inline unsigned CurrEFComposerId = 0; // NOLINT
```

Comment:
What's the issue here?

Reply:
`Variable 'CurrEfComposerId' is non-const and globally accessible, consider making it constclang-tidy(cppcoreguidelines-avoid-non-const-global-variables)`